ferrocene/doc/qualification-plan/src/standards/iec-61508.rst

```diff
@@ -8,127 +8,169 @@ IEC 61508 requirements
    :header-rows: 1
 
    * - Clause
-     - Document chapter(s)
+     - ``rustc``
+     - ``rustfmt``
 
    * - 7.4.4.1
      - N/A
+     - N/A
 
    * - 7.4.4.2
      - N/A
+     - N/A
 
    * - 7.4.4.3
      - This qualification is the justification
+     - This qualification is the justification
 
    * - 7.4.4.4
-     - | :doc:`Safety Manual - Usage <safety-manual:rustc/usage>`
-       | :doc:`Safety Manual - Degraded environment <safety-manual:rustc/degraded-environment>`
-       | :doc:`Evaluation Report - Use cases <evaluation-report:rustc/use-cases>`
+     - | :doc:`Safety Manual - rustc - Usage <safety-manual:rustc/usage>`
+       | :doc:`Safety Manual - rustc - Degraded environment <safety-manual:rustc/degraded-environment>`
+       | :doc:`Evaluation Report - rustc - Use cases <evaluation-report:rustc/use-cases>`
+     - | :doc:`Safety Manual - rustfmt - Usage <safety-manual:rustfmt/usage>`
+       | :doc:`Evaluation Report - rustfmt - Use cases <evaluation-report:rustfmt/use-cases>`
 
    * - 7.4.4.5
-     - :doc:`Evaluation Report - Tool analysis <evaluation-report:rustc/tool-analysis>`
+     - :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustc/tool-analysis>`
+     - :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustfmt/tool-analysis>`
 
    * - 7.4.4.6
-     - | :doc:`Evaluation Report - Tool analysis <evaluation-report:rustc/tool-analysis>`
-       | :doc:`Qualification Report - Test results <qualification-report:rustc/index>`
+     - | :doc:`Evaluation Report - rustc - Tool analysis <evaluation-report:rustc/tool-analysis>`
+       | :doc:`Qualification Report - rustc - Test results <qualification-report:rustc/index>`
+     - :doc:`Evaluation Report - rustfmt - Tool analysis <evaluation-report:rustfmt/tool-analysis>`
 
    * - 7.4.4.7
-     - See items below
+     - See sub items below
+     - See sub items below
 
    * - 7.4.4.7.a
      - :doc:`Qualification Plan - Validation process <qualification-plan:validation>`
+     - N/A
 
    * - 7.4.4.7.b
      - :doc:`Document List <document-list:index>`
+     - N/A
 
    * - 7.4.4.7.c
      - :doc:`Evaluation Report - Use cases <evaluation-report:rustc/use-cases>`
+     - N/A
 
    * - 7.4.4.7.d
      - :doc:`Evaluation Report - Qualification method <evaluation-report:rustc/method>`
+     - N/A
 
    * - 7.4.4.7.e
      - :doc:`Qualification Report - Test results <qualification-report:rustc/index>`
+     - N/A
 
    * - 7.4.4.7.f
      - :doc:`Qualification Report - Test results <qualification-report:rustc/index>`
+     - N/A
 
    * - 7.4.4.7.g
      - :doc:`Qualification Report - Test results <qualification-report:rustc/index>`
+     - N/A
 
    * - 7.4.4.8
      - N/A
+     - N/A
```

Comment:
We should fill this, as we don't have the results from 7.4.4.6 and we instead have effective measures to control failures (manual review).

Reply:
Yes, we can reference the analysis, that all failures will be caught here.

Reply:
Linked the detection measures and usage restrictions